### PR TITLE
add wget dependency to Kraken

### PR DESCRIPTION
**Pull request by hajgato**

### CODE REVIEW

File: easybuild/easyconfigs/w/wget/wget-1.17.1-foss-2016a.eb [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+ the most widely-used Internet protocols. It is a non-interactive commandline tool,
+ so it may easily be called from scripts, cron jobs, terminals without X-Windows support, etc."""
+
+toolchain = {'name': 'foss', 'version': '2016a'}
+
+source_urls = [GNU_SOURCE]
+sources = [SOURCE_TAR_XZ]
+
+dependencies = [
+    ('gnutls', '3.4.11'),
```
Comment (boegel) [inline easybuild/easyconfigs/w/wget/wget-1.17.1-foss-2016a.eb]: same of OpenSSL, let this be provided by the OS? @wpoely86?

Comment (wpoely86) [inline easybuild/easyconfigs/w/wget/wget-1.17.1-foss-2016a.eb]: yes, add osdep but leave the EB dep here but commented out.

**Comment by boegel** [comment]
Test report by @boegel
**FAILED**
Build succeeded for 6 out of 9 (9 easyconfigs in this PR)
node2433.golett.os - Linux centos linux 7.2.1511, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/683076e06ca56de601a1952ade9c59bc for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**FAILED**
Build succeeded for 6 out of 9 (9 easyconfigs in this PR)
node2041.delcatty.os - Linux SL 6.7, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/ec841f52fdc7efb51bb06c0e01552a19 for a full test report.

File: easybuild/easyconfigs/w/wget/wget-1.17.1-foss-2016a.eb [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+    ('zlib', '1.2.8'),
+    #('OpenSSL', '1.0.1s'),  # OS dependency should be preferred if the os version is more recent then this version, it's
+    # nice to have an up to date openssl for security reasons
+]
+
+#Optionally, you can use OpenSSL insted of gnutls
+#configopts = '--with-ssl=openssl '
+#
+#osdependencies = [('openssl-devel', 'libssl-dev', 'libopenssl-devel')]
+osdependencies = [('gnutls')]
```
Comment (boegel) [inline easybuild/easyconfigs/w/wget/wget-1.17.1-foss-2016a.eb]: despite that this is specified, `gnutls` is not found on our systems:

```
configure: error: Package requirements (gnutls) were not met:

No package 'gnutls' found

Consider adjusting the PKG_CONFIG_PATH environment variable if you
installed software in a non-standard prefix.
```

Maybe it's better to go with OpenSSL after all?

Comment (hajgato) [inline easybuild/easyconfigs/w/wget/wget-1.17.1-foss-2016a.eb]: I agree. I think `RHEL`, `SL`, and `COS` use `OpenSSL` for `wget` from repo.

Comment (hajgato) [inline easybuild/easyconfigs/w/wget/wget-1.17.1-foss-2016a.eb]: @boegel ^

Comment (boegel) [inline easybuild/easyconfigs/w/wget/wget-1.17.1-foss-2016a.eb]: OK, then let's use OpenSSL and drop `gnutls`?

Comment (boegel) [inline easybuild/easyconfigs/w/wget/wget-1.17.1-foss-2016a.eb]: @hajgato already done, sorry, I was looking at an outdated diff

**Comment by boegel** [comment]
lgtm

**Comment by boegel** [comment]
Test report by @boegel
**FAILED**
Build succeeded for 8 out of 9 (9 easyconfigs in this PR)
node2159.delcatty.os - Linux SL 6.7, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/5a8b500dcf79915a57a518a3f8953edb for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**FAILED**
Build succeeded for 8 out of 9 (9 easyconfigs in this PR)
node2608.swalot.os - Linux centos linux 7.2.1511, Intel(R) Xeon(R) CPU E5-2660 v3 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/cf774f8ff67d493537aaf452fd2ea64b for a full test report.

**Comment by hajgato** [comment]
@boegel `gnutls` ftp?

**Comment by boegel** [comment]
@hajgato yeah, sorry, fixing that

File: easybuild/easyconfigs/w/wget/wget-1.17.1-foss-2016a.eb (hunk 2 above)
Comment (wpoely86) [inline easybuild/easyconfigs/w/wget/wget-1.17.1-foss-2016a.eb]: @boegel yeah, `gnutls-devel` is not installed (remember my quattor pr?)

Don't drop `gnutls` fully, just comment it out.

Comment (hajgato) [inline easybuild/easyconfigs/w/wget/wget-1.17.1-foss-2016a.eb]: @wpoely86 Check the updated version. osdep `gnutls` updated for various linux (using the develop version)

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 9 out of 9 (9 easyconfigs in this PR)
node2608.swalot.os - Linux centos linux 7.2.1511, Intel(R) Xeon(R) CPU E5-2660 v3 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/d17ccba066bcda02d8d9e6b07e13fad5 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 9 out of 9 (9 easyconfigs in this PR)
node2159.delcatty.os - Linux SL 6.7, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/e7e1551c3ec973829feb3aeedd9aa4ea for a full test report.

**Comment by boegel** [comment]
Going in, thanks @hajgato!

**Pull request merged**